examples/get_merfish_u20s_data.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import io
+import os
+from typing import IO, Tuple
+
+from skimage.io import imread, imsave
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Indices
+from starfish.util.argparse import FsExistsType
+
+SHAPE = 2048, 2048
+
+
+class MERFISHTile(FetchedTile):
+    def __init__(self, file_path, hyb, ch):
+        self.file_path = file_path
+        # how to index tiles from indices into multi-page tiff
```

Comment:
I want to make sure I'm clear on what this means: 
`# maps information on the acquisition channel and round to an index that identifies the correct page in the original multi-page tiff file`

Reply:
Thanks.